pkg/manifestclient/list.go

```diff
@@ -122,9 +128,12 @@ func (mrt *manifestRoundTripper) listAll(requestInfo *apirequest.RequestInfo) ([
 
 func (mrt *manifestRoundTripper) listAllNamespaces() ([]byte, error) {
 	possibleNamespaceFiles, err := allPossibleNamespaceFiles(mrt.sourceFS)
-	if err != nil {
+	switch {
+	case errors.Is(err, fs.ErrNotExist):
+	case err != nil:
```

Comment:
Note to self: previously it'd return the error if it didn't exist, now it does nothing